sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/PendingTransactionDb.kt

```diff
@@ -70,10 +70,10 @@ interface PendingTransactionDao {
     suspend fun removeRawTransactionId(id: Long)
 
     @Query("UPDATE pending_transactions SET minedHeight = :minedHeight WHERE id = :id")
-    suspend fun updateMinedHeight(id: Long, minedHeight: Int)
+    suspend fun updateMinedHeight(id: Long, minedHeight: Long)
 
     @Query("UPDATE pending_transactions SET raw = :raw, rawTransactionId = :rawTransactionId, expiryHeight = :expiryHeight WHERE id = :id")
-    suspend fun updateEncoding(id: Long, raw: ByteArray, rawTransactionId: ByteArray, expiryHeight: Int?)
+    suspend fun updateEncoding(id: Long, raw: ByteArray, rawTransactionId: ByteArray, expiryHeight: Long?)
```

Comment:
Does this imply a migration or it automatically figures it out?

Reply:
SQLite doesn't use strong types, so this should be fine as-is.  https://www.sqlite.org/datatype3.html

Reply:
Oh yeah. that gives me nightmares. I'm in denial 😅